src/main/java/formflow/library/upload/S3CloudFileRepository.java

```diff
@@ -71,6 +75,25 @@ public void upload(String filePath, MultipartFile file) {
     }
   }
 
+  public CloudFile get(String filepath) {
+    try {
+      log.info("Getting file at filepath {} from S3", filepath);
+      S3Object s3Object = s3Client.getObject(bucketName, filepath);
+      S3ObjectInputStream inputStream = s3Object.getObjectContent();
+
+      ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+      IOUtils.copy(inputStream, outputStream);
+      byte[] fileBytes = outputStream.toByteArray();
+      long fileSize = fileBytes.length;
+
+      log.info("File {} successfully downloaded", filepath);
+      return new CloudFile(fileSize, fileBytes);
+    } catch (IOException e) {
+      log.error("Exception occurred while attempting to get file in S3 code: " + e.getMessage());
```

Comment:
I think it's worthwhile to include more information here, unless the e.getMessage() will always include "filepath" in the event of an error? If not, including "filepath" in the log messages is good for debugging purposes.